src/profiling.wat

```diff
@@ -0,0 +1,26 @@
+(module
+  (func $imports.getTime (;0;) (import "imports" "getTime") (result i32))
+  (func $main (;0;) (export "main") (result i32)
+    (local $time i32)
```

Comment:
The `i32` doesn't look safe here. For time stamps since the epoch you probably need an `i64`.

Reply:
True, thanks for catching this. Changed to `i64`!